Nuclei crashes with OpenAPI Bearer input

### Nuclei version: v3.2.4

### Current Behavior:
When I run `nuclei` with an OpenAPI specs, it crashes:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x10 pc=0x1051b67d4]

goroutine 1 [running]:
github.com/projectdiscovery/nuclei/v3/pkg/input/formats/openapi.generateRequestsFromOp(0x14000716e60)
	github.com/projectdiscovery/nuclei/v3/pkg/input/formats/openapi/generator.go:188 +0x374
github.com/projectdiscovery/nuclei/v3/pkg/input/formats/openapi.GenerateRequestsFromSchema(0x14000c825a0, {0x14000b26900?, 0xf?, 0x0?}, 0x14000f1c690)
	github.com/projectdiscovery/nuclei/v3/pkg/input/formats/openapi/generator.go:84 +0x9d4
github.com/projectdiscovery/nuclei/v3/pkg/input/formats/openapi.(*OpenAPIFormat).Parse(0x108a48cb0, {0x16ba122ae, 0xf}, 0x0?)
	github.com/projectdiscovery/nuclei/v3/pkg/input/formats/openapi/openapi.go:38 +0x9c
github.com/projectdiscovery/nuclei/v3/pkg/input/provider/http.NewHttpInputProvider(0x14000faf9d8)
	github.com/projectdiscovery/nuclei/v3/pkg/input/provider/http/multiformat.go:51 +0x17c
github.com/projectdiscovery/nuclei/v3/pkg/input/provider.NewInputProvider({0x108a556c0?, 0x0?})
	github.com/projectdiscovery/nuclei/v3/pkg/input/provider/interface.go:111 +0x230
github.com/projectdiscovery/nuclei/v3/internal/runner.New(0x108a556c0)
	github.com/projectdiscovery/nuclei/v3/internal/runner/runner.go:228 +0x7cc
main.main()
	./main.go:122 +0x33c
```

### Expected Behavior:
`nuclei` runs and generates the requests based on the OpenAPI endpoints list.

### Steps To Reproduce:
1. I am using the following OpenAPI file named `openapi.yml`:
```
openapi: "3.0.0"
info:
  title: "Acme API"
  description: "API documentation for Acme invitation management."
  version: "1.0.0"
servers:
  - url: "https://acme.com/api/v1/rest"
paths:
  /invitations:
    post:
      summary: "Create an invitation"
      description: "Send an invitation with specified details."
      parameters:
        - name: "invitation_type"
          in: "query"
          description: "Type of the invitation."
          required: true
          schema:
            type: "string"
            example: "email"
        - name: "invite_admin"
          in: "query"
          description: "Whether to invite as admin."
          required: true
          schema:
            type: "boolean"
            example: true
      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: "object"
              properties:
                email:
                  type: "string"
                  example: "bva@acme.com"
                invitee_name:
                  type: "string"
                  example: "abc"
                account_type_id:
                  type: "integer"
                  example: 5
      responses:
        "200":
          description: "Invitation created successfully."
          content:
            application/json:
              schema:
                type: "object"
                properties:
                  status:
                    type: "string"
                    example: "success"
                  message:
                    type: "string"
                    example: "Invitation sent successfully."
        "400":
          description: "Bad request if the input parameters are incorrect."
        "401":
          description: "Unauthorized if the authorization token is invalid."
      security:
        - bearerAuth: []
components:
  securitySchemes:
    bearerAuth:
      type: "http"
      name: "Authorization"
      scheme: "bearer"
      bearerFormat: "JWT"
```
2. I am using the following secrets file named `secrets.yml`:
```
# static secrets
static:
  - type: cookie
    domains:
      - acme.com
    cookies:
      - key: token
        value: xxxxx
```
3. I am running `nuclei` with the following command: 
```
nuclei -sf secrets.yml -im openapi -l openapi.yml
```

--- a/pkg/input/formats/openapi/generator.go
+++ b/pkg/input/formats/openapi/generator.go
@@ -179,6 +179,10 @@ func generateRequestsFromOp(opts *generateReqOptions) error {
 	for _, parameter := range reqParams {
 		value := parameter.Value
 
+		if value.Schema == nil || value.Schema.Value == nil {
+			continue
+		}
+
 		// paramValue or default value to use
 		var paramValue interface{}
 

